torch/_dynamo/trace_rules.py

```diff
@@ -3231,6 +3230,17 @@ def _module_dir(m: types.ModuleType):
     "torch._higher_order_ops.strict_mode",
     "torch._higher_order_ops.while_loop",
     "torch._higher_order_ops.associative_scan",
+    "torch.optim.optimizer",
+    "torch.optim.adam",
+    "torch.optim.adamw",
+    "torch.optim.radam",
+    "torch.optim.nadam",
+    "torch.optim.adamax",
+    "torch.optim.rmsprop",
+    "torch.optim.rprop",
+    "torch.optim.asgd",
+    "torch.optim.sgd",
+    "torch.optim.adadelta",
```

Comment:
Is there any files under ```torch.optim``` which we don't want to inline? If no, we can just put one line here.

Reply:
Yeah I don't want to inline the LR schedulers (that's why I did this haha)

Although weirdly it looks like it removed a graph break somehow, I'm going to look into this tomorrow.